fetch.bs

```diff
@@ -3569,6 +3625,9 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
    <li class=XXX><p>Needs testing: multiple `<code>Proxy-Authenticate</code>` headers, missing,
    parsing issues.
 
+   <li><p>If fetch is <a for=fetch>terminated</a>, providing a <var>reason</var>, return a
+   <a>network error</a> with <a for=response>termination reason</a> set to <var>reason</var>.
```

Comment:
then return*

But also, this seems to be in a strange place. Why do we have this for 407, but not 401? There's also no enclosing step saying these steps can be terminated.